examples/integrations/oci/objectstorage-cdi/src/main/java/module-info.java

```diff
@@ -18,18 +18,21 @@
  * Example of integration with OCI object storage in a CDI application.
  */
 module io.helidon.examples.integrations.oci.objectstorage.cdi {
+    requires java.logging;
     requires java.ws.rs;
     requires java.json.bind;
+
     requires jakarta.inject.api;
+    requires jakarta.enterprise.cdi.api;
```

Comment:
I admit that module names have changed during the torturous path toward Jakartafication but carefully check to see whether on the `helidon-2.x` line these shouldn't be `javax.xyz` rather than `jakarta.xyz`.  The important thing, I think, is that whichever modules they are, they have `javax.*` packages, not `jakarta.*` packages.

Reply:
Have removed it.